registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformSuccessMessages.java

```diff
@@ -172,7 +172,10 @@ public enum PlatformSuccessMessages {
 			"Packet processing completed with reject status without Parent flow"),
 
 	PACKET_ANONYMOUS_PROFILE(PlatformConstants.RPR_CAMEL_BRIDGE_MODULE + "008",
-			"Packet anonymous profile flow initiated");
+			"Packet anonymous profile flow initiated"),
+	
+	RPR_MESSAGE_SENDER_API_SUCCESS(PlatformConstants.RPR_USER_NOTIFICATION_API + "000",
```

Comment:
RPR_MESSAGE_SENDER_API_SUCCESS to RPR_USER_NOTIFICATION_API_SUCCESS

Reply:
done, please check earliest commit